check_slack_connectivity.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+import slack_sdk
+
+import slack_reporter_config as config
```

Comment:
any reason we're not doing this by environment variables (like in a `.env` file) instead?

Reply:
We discussed: env vars are super useful when you're running as a CI and you're dynamically provisioning new containers to run in and your secrets are stored in some external thing. but if we're just running on local boards, it's 6 of one, half a dozen of the other. I'm going to leave this as-is, but it might need changing if we redo the architecture of where the canaries run.